mflowgen/common/init-fullchip/outputs/route-bumps.tcl

```diff
@@ -308,21 +306,23 @@ proc fix_jtag {} {
     # Rip up and reroute JTAG wire that otherwise
     # shorts with sjk analog phy routing
 
-    # Delete old route
-    set net pad_jtag_intf_i_phy_tck
-    editDelete -net $net
+    # Clean slate
+    deselectAll
+    
+    # Delete old routes
+    set net pad_jtag_intf_i_phy_tck; editDelete -net $net
+    set net pad_jtag_intf_i_phy_tdi; editDelete -net $net
```

Comment:
Why are these being deleted?

Reply:
The default route put these two wires over a part of the chip that PHY people wanted to use. The fix_jtag hack rips up and reroutes those wires so they go around that area. I updated the comments to try and make that more clear.